pkg/controller/migplan/registry.go

```diff
@@ -186,15 +170,7 @@ func (r ReconcileMigPlan) ensureRegistryDC(client k8sclient.Client, plan *migapi
 }
 
 // Ensure the service for the migration registry on the specified cluster has been created
-func (r ReconcileMigPlan) ensureRegistryService(client k8sclient.Client, plan *migapi.MigPlan) error {
-	registrySecret, err := plan.GetRegistrySecret(client)
-	if err != nil {
-		return err
-	}
-	if registrySecret == nil {
-		return errors.New("migration registry credentials not found")
-	}
-	name := registrySecret.GetName()
+func (r ReconcileMigPlan) ensureRegistryService(client k8sclient.Client, plan *migapi.MigPlan, name string) error {
```

Comment:
So this is no change from current behavior, but essentially that is correct. These resources will all have the same name. This seems to be the general kube convention for this sort of thing. For example, when you expose a route for "my-service", the route is also called "my-service". When adding a service to "my-dc", the service will also be called "my-dc". While the names don't have to be the same, if the names are all different, it's going to be somewhat more error-prone since a number of these resources reference others by name (i.e. "app" labels, etc.). I don't know if it's any riskier than other cases where service names match the names of their corresponding resources.

Reply:
I can't really think of any situation where the service names could clash, since the only time we ever use the uniquely-determined secret name as a service is right after creating (or pulling) the associated secret.

Reply:
Ok, yeah I agree it seems like low possibilty of collision. I think there are ~`60,466,176` possible names here. We can address later if it's an issue.